src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ChoiceWatchRunListener.java

```diff
@@ -42,13 +42,8 @@
     @Override
     public void onFinalized(AbstractBuild<?,?> build)
     {
-        Object p = build.getProject();
+        AbstractProject<?,?> job = build.getProject();
         
-        if(!(p instanceof AbstractProject<?,?>))
-        {
-            return;
-        }
-        AbstractProject<?,?> job = (AbstractProject<?,?>)p;

```

Comment:
<file classname='jp.ikedam.jenkins.plugins.extensible_choice_parameter.ChoiceWatchRunListener'>
<BugInstance type='BC_VACUOUS_INSTANCEOF'
priority='Normal'
category='STYLE'
message='instanceof will always return true for all non-null values in jp.ikedam.jenkins.plugins.extensible_choice_parameter.ChoiceWatchRunListener.onFinalized(AbstractBuild), since all hudson.model.AbstractProject are instances of hudson.model.AbstractProject'
lineNumber='47'/>
</file>